routers/web/repo/view.go

```diff
@@ -578,6 +586,15 @@ func renderFile(ctx *context.Context, entry *git.TreeEntry, treeLink, rawLink st
 		}
 	}
 
+	if fInfo.st.IsImage() && !fInfo.st.IsSvgImage() {
+		img, _, err := image.DecodeConfig(bytes.NewReader(buf))
+		if err == nil {
+			// There are Image formats go can't decode
+			// Instead of throwing an error in that case, we show the size only when we can decode
+			ctx.Data["ImageSize"] = fmt.Sprintf("%dx%dpx", img.Width, img.Height)
```

Comment:
why ' !fInfo.st.IsSvgImage()' not svg?

Reply:
svg is text-based instead of binary, and vector-based instead of line-by-line, so it doesn't really have a "size".
It is only limited by the view box that can be arbitrarily scaled.

Reply:
Not entirely correct. SVGs can have `height` and `width`, and if they are absent, the size of the viewport is used IIRC. If even that is absent, browser will use `300×150` IIRC. Still I guess it's enough if this PR handles only raster images for now.